meeting/urls.py

```diff
@@ -10,18 +10,31 @@
 urlpatterns = [
     # Home Page
     path('', views.index, name='index'),
+    
+    #Availability
     path('availability', availability.availability, name='Availability'),
     path('availability/<int:meeting_id>', availability.Availability.as_view(), name='meeting_availability'),
     path('availability/<int:meeting_id>/delete', availability.AvailabilityDelete.as_view(), name='AvailabilityDelete'),
-    path('login', authentication.LoginProcess.as_view(), name="LoginProcess"),
+    
+    #Notification
     path('notification_demo', notifications.notification_demo, name='notification_demo'),
     path('notifications', notifications.NotificationProcess.as_view(), name='NotificationProcess'),
+    
+    #User
+    path('login', authentication.LoginProcess.as_view(), name="LoginProcess"),
     path('profile', authentication.ProfilePage.as_view(), name="ProfilePage"),
+    path('register', authentication.RegisterProcess.as_view(), name="RegisterProcess"),
+    
+    #Project
     path('projects', projects.projects, name='projects'),
     path('projects/create', projects.ProjectCreationProcess.as_view(), name='ProjectCreationProcess'),
     path('projects/<int:project_key>', projects.ProjectViewProcess.as_view(), name="ProjectViewProcess"),
     path('projects/<int:project_key>/edit', projects.ProjectModificationProcess.as_view(), name="ProjectModificationProcess"),
+    
+    #Meeting
     path('projects/<int:project_key>/meetings/create', meetings.MeetingView.as_view(), name="MeetingView"),
-    path('register', authentication.RegisterProcess.as_view(), name="RegisterProcess"),
-    path('voting', voting.Voting.as_view(), name='Voting')
+    path('projects/<int:meeting_key>/meetings/view', meetings.MeetingEdit.as_view(), name="MeetingEdit"),
```

Comment:
This breaks the "view meeting" button on the projects page. However, since that button is mine I can merge this pull request and offer a patch later. It might be good for us to change that URL to something like '/meetings/<int:meeting_key>' to avoid confusion--it doesn't matter a whole lot for this project, but most API developers would look at the `projects/<key>` piece and assume that the `<key>` was a project ID, even though it's named differently.

Reply:
Yeah, all of that is actually buggy. I should've changed the name back though before making the pr because the value is still the project ID. That's my bad.